### PR TITLE
AddingPowerSupport_CI/Testing

### PR DESCRIPTION
**Pull request by santosh653**
Adding power support.,

Adding power support ppc64le with Continues Integration/testing so that code remains architecture independent.

This is part of the Ubuntu distribution for ppc64le. This helps us simplify testing later when distributions are re-building and re-releasing. For more info tag @gerrith3.

The build is successful on both arch: amd64/ppc64le, please find the Travis Link below.
https://travis-ci.com/github/santosh653/sftp

Please let me know if you need any further details.

Thank You !!
@santosh653

### CODE REVIEW

**Review by drakkan — APPROVED** [review]

**Comment by drakkan** [comment]
LGTM, maybe we should also enable testing on arm64 in a separate commit

**Comment by drakkan** [comment]
In general I'm not very happy with travis, it is really slow. For example I submitted a PR more than an hour ago and the build is not started yet. I would like to switch to GitHub actions

**Review by drakkan — CHANGES_REQUESTED** [review]
Before merging this PR I suggest to decide if accept #402 or continue to use travis

**Comment by santosh653** [comment]
@drakkan Thanks for your updates, As for now we will continue to use Travis. Not sure but Builds on Travis look ok in terms of running time barring couple of instances.
Hope this answers your query.
Thank You !!

**Comment by drakkan** [comment]
> @drakkan Thanks for your updates, As for now we will continue to use Travis. Not sure but Builds on Travis look ok in terms of running time barring couple of instances.
> Hope this answers your query.
> Thank You !!

@santosh653 the question was not for you but for other pkg/sftp maintainers, sorry for the confusion. 

Currently using travis is really unpleasant. For example today I submitted a PR that introduced a regression on macOS, I had to wait for an hour or more for Travis to build my changes. I added a github action to my repo and fixed the bug much faster.

So we should switch to travis.com or to github actions

**Comment by gerrith3** [comment]
@drakkan you definitely want to get off of travis-ci.ORG - they announced they were deprecating it about two years ago.  They are now slowing down jobs there to send the message to move.   Not sure if the message is clear enough, but there are no delays in starting jobs on travis-ci.COM unless there is a serious overload somewhere (which I doubt).

**Comment by drakkan** [comment]
> @drakkan you definitely want to get off of travis-ci.ORG - they announced they were deprecating it about two years ago. They are now slowing down jobs there to send the message to move. Not sure if the message is clear enough, but there are no delays in starting jobs on travis-ci.COM unless there is a serious overload somewhere (which I doubt).

Yes, as said above and in #402 we should switch to travis-ci.com or github actions. I think we have to wait for the opinion of other pkg/sftp maintainers as well, #402 is the right issue for discussing about this, thank you for your patience

**Comment by puellanivis** [comment]
:woman_shrugging: sounds like Github actions might be the best choice going forward.

**Comment by santosh653** [comment]
Are there any queries open on this PR?

**Comment by drakkan** [comment]
> Are there any queries open on this PR?

No thank you for your effort. 

We (authors and maintainers of pkg/sftp) should decide whether to accept this PR and continue using travis (switching to travis-ci.com) or reject the PR and move on to GitHub actions, thanks for your patience

**Comment by santosh653** [comment]
Do we have any further queries on this PR? Is this PR getting merged?

**Comment by drakkan** [comment]
> Do we have any further queries on this PR? Is this PR getting merged?

Hi,

I would prefer to merge #402, I added support to run tests on aarch64, ppc64le and armv7

**Comment by puellanivis** [comment]
I suppose that I am fine with either. We just need to decide.

**Comment by drakkan** [comment]
Hi, we switched to GitHub Actions so this PR cannot be merged, sorry

**Pull request closed without merging**
